include/sys/spa.h

```diff
@@ -18,16 +18,19 @@
  *
  * CDDL HEADER END
  */
+ 
 /*
  * Copyright (c) 2005, 2010, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2011, 2019 by Delphix. All rights reserved.
- * Copyright 2011 Nexenta Systems, Inc.  All rights reserved.
- * Copyright (c) 2014 Spectra Logic Corporation, All rights reserved.
- * Copyright 2013 Saso Kiselkov. All rights reserved.
- * Copyright (c) 2014 Integros [integros.com]
- * Copyright 2017 Joyent, Inc.
- * Copyright (c) 2017 Datto Inc.
- * Copyright (c) 2017, Intel Corporation.
+ * Copyright (c) 2011, 2019 Delphix. All rights reserved.
+ * Copyright (c) 2011, Nexenta Systems, Inc.  All rights reserved.
+ * Copyright (c) 2014, Spectra Logic Corporation, All rights reserved.
+ * Copyright (c) 2013, Saso Kiselkov. All rights reserved.
+ * Copyright (c) 2014, Integros. All rights reserved.
+ * Copyright (c) 2017, Joyent, Inc. All rights reserved.
+ * Copyright (c) 2017, Datto Inc. All rights reserved.
+ * Copyright (c) 2017, Intel Corporation. All rights reserved.
+ * Copyright (c) 2019, Klara Inc. All rights reserved.
```

Comment:
same as above

Reply:
It's debatable if this one line is copyrightable.
`#define	SPA_COMPRESSMASK	((1U << SPA_COMPRESSBITS) - 1)`

I think it's more than a obvious fix, as it requires a relatively high amount of creativity and is meant to add a additional feature.

Didn't want to get in any debate, if you're not sure if it's copyrightable assume it is.

Reply:
ack